### DISCUSSION TITLE
Is it possible to use http in the repositories?

### DISCUSSION THREAD

**Discussion by root3200** [semaphoreui/semaphore#2041] [Q&A] [answered]
hello community, is it possible to use the http protocol in the repositories ?

I have a private repository, but I can't use it because ansible-semaphore forces me to use https.

![Captura de pantalla 2024-05-22 205700](https://github.com/semaphoreui/semaphore/assets/58263431/ea895eb2-0df4-4d4e-9476-772529c3e6a6)

**Comment by fiftin** [comment, marked as answer]
Hi, it is UI issue. Will fix ASAP.

**Reply by root3200** [reply]
ok, thanks for your answer, I would appreciate that in the next release you comment the change.
